annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -229,17 +243,11 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
         return getHotspotAnnotation(variants);
     }
 
-    @ApiOperation(value = "Retrieves mutation assessor information for the provided list of variants",
-        nickname = "getMutationAssessorAnnotation")
-    @ApiResponses(value = {
-        @ApiResponse(code = 200, message = "Success",
-            response = MutationAssessor.class,
-            responseContainer = "List"),
-        @ApiResponse(code = 400, message = "Bad Request")
-    })
-    @RequestMapping(value = "/mutation_assessor/{variants:.+}",
-        method = RequestMethod.GET,
-        produces = "application/json")
+//    @ApiOperation(value = "Retrieves mutation assessor information for the provided list of variants",
```

Comment:
same comment as previous

Reply:
A separate internal `MutationAssessorController` is on the way.